sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JDBCRelation.scala

```diff
@@ -90,6 +90,21 @@ private[sql] case class JDBCRelation(
 
   override val schema: StructType = JDBCRDD.resolveTable(url, table, properties)
 
+  /**
+   * Check if [[JDBCRDD.compileFilter]] can accept input filters.
```

Comment:
According to [guideline](http://docs.scala-lang.org/style/scaladoc.html), it looks classes should be separately wrapped with [[ ]] :).